app/javascript/initializeGame.js

```diff
@@ -15,9 +16,12 @@ angular.module('dappChess').controller('InitializeGameCtrl',
       $rootScope.$broadcast('message', 'Your game is being created, please wait a moment...',
                             'loading', 'startgame');
       try {
-        
+
         Chess.initGame($scope.username, $scope.startcolor === 'white',
-                     { from: $scope.selectedAccount });
+         {
+           from: $scope.selectedAccount,
+           value: web3.toWei($scope.etherbet.replace(',', '.'), 'ether')
```

Comment:
`$scope.etherbet` should be checked. It is a `type="text"` field and the user can enter arbitrary things ;)